src/forms/Textarea.jsx

```diff
@@ -66,26 +65,6 @@ export class Textarea extends React.PureComponent<Props, State> {
 		return overrideValue(nextProps);
 	}
 
-	/**
-	 * Turns on autosize if requested
-	 * @return {undefined} side effect only
-	 */
-	componentDidMount() {
-		if (this.props.autosize) {
```

Comment:
Does removing autosize break mup-web behavior? I'm wondering if we should just leave it in but no pass through to swarm ui?